app/templates/views/guidance/pricing/how-to-pay.html

```diff
@@ -58,7 +58,7 @@ <h2 class="heading-medium" id="purchase-orders">
        <li>a purchase order (PO) for the financial year</li>
        <li>confirmation your organisation can receive invoices without a PO reference</li>
 </ul>
-<p class="govuk-body"><a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.support')}}">Send the PO or confirmation using our support form</a>.</p>
+<p class="govuk-body">Send your <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.support')}}">PO or confirmation with the name and email address of your finance contact using our support form</a>.</p>
```

Comment:
Is this link text maybe a bit on the long side?

If it is, we could try:

```suggestion
<p class="govuk-body">Send your PO or confirmation with the name and email address of your finance contact using <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.support')}}">our support form</a>.</p>
```

Reply:
I did think about this and thought it was important to add the action incase the user misses it. Aesthetically it looks way better as just the 'our support form'.

Reply:
Screenshot from this running locally (on Google Chrome) on desktop, in case it helps:
<img width="638" alt="image" src="https://github.com/alphagov/notifications-admin/assets/87140/3f69cf07-2a4d-42cc-97ef-e04eb9cef663">

Reply:
Thanks Tom.

I don’t feel too strongly about this - but (as you mention, Saima) it would be good to include the action as part of the link.

What about something like:

```suggestion
<p class="govuk-body">Use our support form to <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.support')}}">send us your PO or confirmation with the name and email address of your finance contact</a>.</p>
```

Reply:
Perfect!